kittycad/client_test.py

```diff
@@ -106,3 +106,54 @@ async def test_file_convert_stl_async():
     assert fc.id is not None
 
     print(f"FileConversion: {fc}")
+
+
+def test_file_mass():
+    # Create our client.
+    client = ClientFromEnv()
+
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    file = open(os.path.join(dir_path, "../assets/testing.obj"), "rb")
```

Comment:
Test for both mass and volume fail when I tried to use the original `testing.stl` so I added this obj and these tests are happy now.

The error when using the stl is
```bash
Error(error_code='InvalidValue', message='unsupported value for "body": could not get mass: expected mesh for current workaround', request_id='6d1aaf21-caf8-49db-9856-7c8c7395b797', additional_properties={})
```
https://github.com/KittyCAD/kittycad.py/runs/6879467418?check_suite_focus=true

I assuming the problem is more likely to be in the engine right?
@hanbollar might this be just a bad mesh with hole in it or something?